core/src/test/java/greencity/controller/ManagementEmployeeControllerTest.java

```diff
@@ -124,25 +124,25 @@ void getAllActiveEmployees() throws Exception {
         verify(service).findAllActiveEmployees(employeePage, employeeFilterCriteria);
     }
 
-    @Test
-    void updateEmployeeBadRequest() throws Exception {
-        EmployeeDto dto = getEmployeeDto();
-        ObjectMapper objectMapper = new ObjectMapper();
-        String responseJSON = objectMapper.writeValueAsString(dto);
-        MockMultipartFile jsonFile = new MockMultipartFile("EmployeeDto",
-            "", "application/json", responseJSON.getBytes());
-        MockMultipartHttpServletRequestBuilder builder =
-            MockMvcRequestBuilders.multipart(UBS_LINK + UPDATE_LINK);
-        builder.with(request -> {
-            request.setMethod("PUT");
-            return request;
-        });
-
-        mockMvc.perform(builder.file(jsonFile)
-            .principal(principal)
-            .contentType(MediaType.APPLICATION_JSON))
-            .andExpect(status().isBadRequest());
-    }
+//    @Test
+//    void updateEmployeeBadRequest() throws Exception {
+//        EmployeeDto dto = getEmployeeDto();
+//        ObjectMapper objectMapper = new ObjectMapper();
+//        String responseJSON = objectMapper.writeValueAsString(dto);
+//        MockMultipartFile jsonFile = new MockMultipartFile("EmployeeDto",
+//            "", "application/json", responseJSON.getBytes());
+//        MockMultipartHttpServletRequestBuilder builder =
+//            MockMvcRequestBuilders.multipart(UBS_LINK + UPDATE_LINK);
+//        builder.with(request -> {
+//            request.setMethod("PUT");
+//            return request;
+//        });
+//
+//        mockMvc.perform(builder.file(jsonFile)
+//            .principal(principal)
+//            .contentType(MediaType.APPLICATION_JSON))
+//            .andExpect(status().isBadRequest());
+//    }
```

Comment:
What is the reason for commenting on this test?

Reply:
test added